### PR TITLE
326: Freezing game state during arbitrage

### PR DESCRIPTION
**Pull request by pimotte**
I chose to do this in a separate rule in order to fix both Trials and Judgements in the same blow. 

作文が好きです、短いだから。= "I like this writing, because it is short."

### CODE REVIEW

**Comment by MrHug** [comment]
Hmm, well how about instead of always having this, you simply say that the prosecution, defendant or person invoking judgement _may_ freeze the gamestate in accordance with this rule?

**Comment by pimotte** [comment]
Safety and simplicity. We already have the vote-override if we really need to, and otherwise there is no problem waiting a day or two. Discussion and voting is still allowed so there's not that much delay either.

**Comment by MrHug** [comment]
True, but in a way a side-effect of this rule is that every Trial must end. I am not saying I find this either a good or a bad thing, but it is a change that we ought to consider as well ;)

**Comment by pimotte** [comment]
I've reformulated a little bit to be able to ignore problematic judgements/trials.

File: README.md [hunk 1]
```diff
@@ -286,8 +286,9 @@ defined as follows.
 **326** As long as an issue labelled "Trial" or "Judgement" exists, no player may start a turn, regardless of what
 other rules specify. Futhermore, any rule specifying that a pull request should be merged in a timely fashion shall
 be read as specifying that the pull request should be merged in a timely fashion, after all issues labelled "Trial"
-or "Judgement" are closed. Any player may ignore this rule, if and only if they have asked for permission by
-creating an issue and a simple majority of players is in favour of allowing this rule to be ignored.
+or "Judgement" are closed. Any player may start a vote in a separate issue to ignore a particular issue labelled 
+"Trial" or "Judgement" for purposes of this rule. 
+That issue will be ignored if a simple majority votes in favour of doing so.
```
Comment (MrHug) [inline README.md:363]: "a simple majority of eligible voters"

EDIT: Or would you say that is implied?

Comment (pimotte) [inline README.md:363]: Implied, and this allows for stuff like abstaining (as in: "count-this-vote-as-if-i-don't-play")

**Comment by MrHug** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by arthurbik** [comment]
I could wait and misuse this, but..

> As long as an issue labelled "Trial" or "Judgement" exists,

This includes closed issues.....

**Comment by pimotte** [comment]
That was definitely what I had in mind and this was a slip of the keyboard.

**Comment by MrHug** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by jdonkervliet** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by arthurbik** [comment]
:+1:

**Comment by Jason-Statham** [comment]
MrHug voted in favor.
jdonkervliet voted in favor.
arthurbik voted in favor.
That is 3 out of 4 players. This PR should be merged. I cannot do that yet.
happy

**Comment by pimotte** [comment]
4 out of 4 votes. Yay:D

**Pull request merged**